File: lintcode/1578.number-of-a/number-of-a.java
public class Solution {
    /**
     * @param s: the given string
     * @return: the number of A
     */
    public int countA(String s) {
        int res = 0;
        for (int i = 0; i < s.length(); i++) {
            if (s.charAt(i) == 'A')
                res++;
        }
        return res;
    }
}

File: lintcode/1578.number-of-a/number-of-a.go
/**
 * @param s: the given string
 * @return: the number of A
 */
 func countA (s string) int {
    res := 0
    for _,v := range s {
        if v == 'A' {
            res++
        }
    }
    
    return res
}
